src/app/backend/client/manager.go

```diff
@@ -110,6 +114,21 @@ func (self *clientManager) APIExtensionsClient(req *restful.Request) (apiextensi
 	return self.InsecureAPIExtensionsClient(), nil
 }
 
+// PluginClient returns a plugin client. In case dashboard login is enabled and
+// option to skip login page is disabled only secure client will be returned, otherwise insecure
+// client will be used.
+func (self *clientManager) PluginClient(req *restful.Request) (pluginclientset.Interface, error) {
```

Comment:
I am wondering how RBAC management for CRDs will work. Will logged in user require privileges to get/list Plugin CRD in order to see the plugins? Should allow plugins to be listed always and only enforce privileges if they will i.e. ask for a list of pods but the user will not have privileges to get them?

Reply:
I think we can move forward with the current changes for now. It may be desirable to not show the available plugins to all users. WDYT?

Reply:
We'll see how it impacts users. We can move forward for now.